src/Common/src/System/Security/Cryptography/DSASecurityTransforms.cs

```diff
@@ -0,0 +1,397 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Is all of this common code new, or was it just moved from somewhere else?  I was assuming the latter, but I don't see any commensurate deletes elsewhere in the PR.  Don't we already have a DSA implementation?